.env.example

```
@@ -0,0 +1,30 @@
+# Readied — Environment Variables
+# Copy this file to .env and fill in the values.
+# The desktop app runs fully offline — these are only needed for the API and web app.
+
+# ─── API (Cloudflare Workers) ───────────────────────────────
+# Set via `wrangler secret put <KEY>` for deployed environments.
+# For local dev, create packages/api/.dev.vars with these values.
+
+TURSO_DATABASE_URL=libsql://your-db.turso.io
+TURSO_AUTH_TOKEN=
+JWT_SECRET=                    # openssl rand -base64 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)